### PR TITLE
Noise profile: Olympus OM-D E-M1 Mk.II (whole ISOs only)

### PR DESCRIPTION
**Pull request by DawidLoubser**
Hi, noise profile for Olympus OM-D E-M1 Mk.II create [as per instructions on this page](http://www.darktable.org/2012/12/profiling-sensor-and-photon-noise/) - tested on a whole range of my images, seems to work extremely well.

Happy to provide the ~6MB Gzip file that the profiling tool products, but don't know where I should send it to?

### CODE REVIEW

**Comment by LebedevRI** [comment]
To https://redmine.darktable.org/projects/darktable/issues/new
We need to see the plots. Else it is impossible to say whether the profile is good or not.

**Comment by DawidLoubser** [comment]
[Dome](https://redmine.darktable.org/issues/11529)

File: data/noiseprofiles.json [hunk 1]
```diff
@@ -3043,7 +3043,22 @@
...
+            {"name": "E-M1MarkII iso 800", "iso": 800, "a": [2.11892459998804e-05, 1.12541173528155e-05, 2.56595516307882e-05], "b": [4.04565499925071e-08, 2.69246669230953e-08, 4.18664327519833e-08]},
+            {"name": "E-M1MarkII iso 1600", "iso": 1600, "a": [4.22953887838241e-05, 2.14948866100995e-05, 4.90974948586661e-05], "b": [8.22142492924679e-08, 6.15912238619032e-08, 8.30371607130872e-08]},
+            {"name": "E-M1MarkII iso 3200", "iso": 3200, "a": [8.09094889300661e-05, 4.03657735201826e-05, 9.11000593033239e-05], "b": [1.54850201636701e-07, 1.06491270486505e-07, 1.4547431044408e-07]},
+            {"name": "E-M1MarkII iso 6400", "iso": 6400, "a": [0.000162818243811292, 7.66465569572085e-05, 0.000189807939134868], "b": [3.4511459701267e-07, 2.72253170538817e-07, 3.08685445026257e-07]},
+            {"name": "E-M1MarkII iso 12800", "iso": 12800, "a": [0.000341007611560922, 0.000170115910815632, 0.000400188562023328], "b": [7.56753327460077e-07, 7.05251299439448e-07, 6.39453192395373e-07]}
+          ]
         }
+
```
Comment (schenlap) [inline data/noiseprofiles.json]: please remove extra newline. Afterward PR can be merges, noise profile looks ok.

Comment (LebedevRI) [inline data/noiseprofiles.json]: @schenlap thanks for checking!

**Comment by LebedevRI** [comment]
@DawidLoubser Hi!
Do you still have `Olympus OM-D E-M1 Mk.II` camera?
I'd love to get the full sample set from that camera for [RPU](https://pixls.us/blog/2017/09/keep-the-raws-coming/).
If you can help, it would be **awesome** if you could take all the following shots, and upload them to [RPU](https://raw.pixls.us/)

Looking at the pdf usermanual, it seems it can produce more than one different raw output (`Record mode and file size/ number of storable still pictures`).
The camera needs to be stable (tripod), in the landscape orientation, and the scene should be as static as possible.

`Record mode` that are interesting:
* `RAW` - should produce one `.ORF` raw
* `25M F+RAW` - not sure what to expect, might produce 4x`.ORI` and then 1x`.ORF`
* `50M F+RAW` - not sure what to expect, might produce 8x`.ORI` and then 1x`.ORF`

I do **NOT** care for the jpeg's it will produce, just ignore them, but do upload *all* the raws.
I will weed the uninteresting duplicates.
Preferably, rename them appropriately before uploading :)

**Comment by atarijedi** [comment]
I have an EM1.2, and I took some raw photos in the various modes.

In terms of what kinds of images it can output, there is the standard RAW .ORF. In both the HIRES modes it creates an 80MP RAW file (.ORF) as well as a .ORI file, then either a 50MP JPG, or a 25MP JPG, the 25MP JPG is like Pentaxs sensor shift, in that each image pixel has real RGB values from each sensor pixel.

I know you said ignore the JPGs, but I included them in the zip file anyways, just in case. I also included a photo of an XRite Color Checker, just in case you wanted to do some colour stuff. 

I am new to github, I tried to include the file in this comment, but its rather large (230MB). I signed up to that redmine page, but I don't know where to go from there?

Let me know where to put the files, and if you want more.

**Comment by LebedevRI** [comment]
@atarijedi please upload *each separate image* excluding JPG's to the https://raw.pixls.us/ !

**Comment by atarijedi** [comment]
Okay, I uploaded 5 files. One standard raw photo of an XRite colour checker, and then 2 raw files for the 50M mode, and 2 raw files for the 25M mode. The HiRes are of a autoparts box and some chart for bicycle parts. If you'd like HiRes photos of something else, like the colour checker, let me know.

**Comment by LebedevRI** [comment]
@atarijedi thanks, but i'm afraid that is not *quite* what i need :(
The normal raw (Olympus_EM1mk2_Standard_20MP_ColorChecker.ORF) has different content from the other raws :( I need *all* the raws to have **exact the same** content.
So if you could, *please* redo all 5 shots :/
(it might be better if it is some outside nature landscape, but this is good too.)

**Comment by atarijedi** [comment]
I just added a standard 20MP raw that is the same as the HiRes files

**Comment by LebedevRI** [comment]
Ok then, thank you, validated!

**Pull request merged**